layouts/partials/header.html

```
@@ -1,3 +1,5 @@
+{{ partial "search-modal.html" . }}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)